senderworker/src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
+
+  <contextName>sender-worker</contextName>
+
+  <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>logs/sender-worker.log</file>
```

Comment:
As a small aside (just in case other people look at this PR), in other logback configs we've specified the log file destination as `logs/application.log`, which I think then makes the script in the user data a little easier to follow. For example, I've been wondering why the `appName` was repeated, but I guess it's because we've named the log file the same as the `appName`,

eg
`/opt/aws-kinesis-agent/configure-aws-kinesis-agent ${this.region} mobile-log-aggregation-${this.stage} /var/log/${props.appName}/${props.appName}.log`

becomes:
`/opt/aws-kinesis-agent/configure-aws-kinesis-agent ${this.region} mobile-log-aggregation-${this.stage} /var/log/${props.appName}/application.log`

Reply:
Thanks David.  This sounds good to me, as the `appName` has already been in the directory name.  It also makes the log file name consistent with other services in `n10n`.